content/working_with/console/widgets/sitesMap.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: bt_wiki
 title: Sites Map
-category: Cloudify Console
+category: Widgets
 draft: false
 ---
 Displays the world map with defined sites marked. Only sites with defined location are displayed on the map.
 
 ![sitesMap]( /images/ui/widgets/sitesMap.png )
 
 {{% note %}}
-The widget is available only when Premium or Spire license is activated on the Cloudify Manager.
+This widget is available with Premium or Spire license.
```

Comment:
Shouldn't we have here `cfy_spire_name` for Spire?

Reply:
I would like to do a second round of fixes for this type of changes.